src/components/ChoiceGroup/ChoiceGroup.css

```diff
@@ -0,0 +1,31 @@
+/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
+@import 'src/css/variables/colors.css';
+@import 'src/css/variables/fonts.css';
+
+.y-choice-group .ms-ChoiceField {
+  margin-bottom: 1.2rem;
+  min-height: 0;
+}
+
+.y-choice-group .ms-ChoiceField-field,
+.y-choice-group .ms-ChoiceField-field::before {
+  border-color: $borderColor;
+}
+
+.y-choice-group.ms-Label {
+  @extend text__mediumSub;
+  font-weight: $fontWeight__bold;
+  color: $textColor__secondary;
+  padding-bottom: 0.4rem;
+}
+
+/* radio button border */
+.y-choice-group .is-checked.ms-ChoiceField-field::before {
+  border-width: 2px;
+  border-color: $borderColor__focused;
+}
+
+/* inside radio button */
+.y-choice-group .ms-ChoiceField-field::after {
+  border-color: $choiceGroupRadioButtonCheck;
+}
```

Comment:
We're moving away from CSS overrides. You can see an example of how to achieve customization without CSS here: https://github.com/Microsoft/YamUI/pull/338/files?short_path=3a4a6ef#diff-00f5ac044af8cd3e73b8070a74488b02R1

Reply:
According to that anatomy of a component thing:

>  A component should consist of these files:
> ComponentName.styles.ts

but i cant find it for ChoiceGroup - am I missing something?